js/index.js

```diff
@@ -39,23 +39,62 @@ function enterPressed (gameState) {
         resultMessage(result)
     }
 }
-
+console.log(gameState.expectedWord)
 function checkWord(gameState) {
     let correctLetters = 0
     let correctPositions = 0
-    for (let i = 0; i < gameState.attemptedWord.length; i++) {
-        if (gameState.attemptedWord[i] === gameState.expectedWord[i]) {
+    gameState.expectedWord = ['E','N','T','E','R']
+    // console.log(gameState.expectedWord)
+    let expectedWordCopy = gameState.expectedWord.slice()
+console.log(expectedWordCopy, 'pre any loop')
+    for (let i = 0; i < 5; i++) {
+        const tile = document.querySelector(`.row${gameState.attemptCounter}` + ' .tile' + i)
+        // const onscreenTile = document.querySelector('.key' + gameState.attemptedWord[i])
+        if (gameState.attemptedWord[i] === expectedWordCopy[i]) {
             correctPositions++
+            expectedWordCopy[i] = ' '
+            const onscreenKey = document.querySelector('.' + gameState.attemptedWord[i].toLowerCase())
+            gameState.attemptedWord[i] = '0'
+            tile.classList.add('correct-position')
+            onscreenKey.classList.add('correct-position')
         }
-        if (gameState.expectedWord.includes(gameState.attemptedWord[i])
-            && gameState.attemptedWord[i] !== gameState.expectedWord[i]) {
+
+    }
+    console.log(expectedWordCopy, 'post correct position loop')
+    for (let g = 0; g < 5; g++) {
+        // onscreenTile.classList.add('correct-position')
+        // const tile = document.querySelector(`.row${gameState.attemptCounter}` + ' .tile' + g)
+        for (let k = 0; k < 5; k++) {
+            const tile = document.querySelector(`.row${gameState.attemptCounter}` + ' .tile' + k)
+            if (expectedWordCopy[g] === gameState.attemptedWord[k]) {
+            console.log(expectedWordCopy, g, gameState.attemptedWord ,k)
+            const onscreenKey = document.querySelector('.' + gameState.attemptedWord[k].toLowerCase())
+            // if (gameState.attemptedWord.includes(expectedWordCopy[g])) {
+            // j = expectedWordCopy.lastIndexOf(gameState.attemptedWord[g]
+            // console.log(j)
             correctLetters++
+            expectedWordCopy[g] = ' '
+            onscreenKey.classList.add('correct-letter')
+            // console.log(expectedWordCopy, g, k)
+
+            tile.classList.add('correct-letter')
```

Comment:
same thing about a const for correct-letter

Reply:
done